infrastructure/pcluster-manager.yaml

```diff
@@ -25,6 +25,20 @@ Parameters:
     Description: Version of AWS ParallelCluster to deploy
     Type: String
     Default: 3.1.2
+  ImageBuilderVpcId:
+    Description: Select the VPC to use for building the container images.
+    Type: AWS::EC2::VPC::Id
+    MinLength: 1 # Force this to be selected
+  ImageBuilderSubnetId:
+    Description: Select the subnet to use for building the container images.
+    Type: AWS::EC2::Subnet::Id
+    MinLength: 1 # Force this to be selected
+
+Conditions:
```

Comment:
This will never be `""` now right?

Reply:
Good call. Conditions removed. Running another launch test for sanity checks.